.github/ISSUE_TEMPLATE/how_to_question.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Question
+about: ❓ How to questions with scanapi
+labels: question, support
+assignees: 
+
+---
+
+### Environment
+- Operating System:
+- Python version:
+
+### What you intend to do with scanapi ?
+<!-- A clear and concise description of what you want to with scanapi. -->
+
+### Expected behavior ?
+<!-- A clear and concise description of what you expected to happen. -->
+
+
+---
+
+scanapi has a [discord server](#) so feel free to join and say 👋 hello. Maybe share the issue link link? 
```

Comment:
@Pradhvan I've created an invite link that never expires, so we can use it: `https://discord.gg/f3cCXJ2ZPB`

Reply:
Great! I Will update that. 😄